### ISSUE TITLE
cli option to rm devenv files from directory

### ISSUE THREAD

**Issue by stuzenz** [cachix/devenv#1350]
devenv is getting better and better - congrats on an amazing project.

A fairly nuanced enhancement (and not high in priority).

I just cloned a project, and was in the mindset of building out the project environment with `devenv init` - I forgot to move into the cloned directory first.

It would be a nice UX feature to have a `devenv rm .` type of command available to remove any remnants of devenv from a directory for scenarios like this.

Thanks for all the great work.

**Comment by domenkozar** [comment]
I don't think this deserves it's own command, shouldn't happen too often I hope :)
